src/logger/logger.class.ts

```diff
@@ -1,20 +1,20 @@
 import { Level } from './level'
 import { Format, FormatOptions } from './format'
-import { createVault, VaultOptions, IVault, Vault } from './vault'
-import { ILogger } from './logger.interface'
+import { createVault, VaultOptions, Vault, VaultKind } from './vault'
+import { Logger } from './logger.interface'
 import { LogContext } from './log-context.class'
 
-export type LoggerOptions = {
+export type LoggerBaseOptions = {
```

Comment:
Just because the 'Logger' has been renamed to 'LoggerBase' does that mean that this has to be renamed too?
The name 'LoggerOptions' still describe what it is used for, with or without the 'Base' part of the name.

Reply:
I would say yes. This is options to the implementation of the `LoggerBase`. `Logger` is an interface and does have options.